web/frontend/tests/forms.spec.ts

```diff
@@ -74,3 +81,77 @@ async function assertIsOnlyVisibleToOwnerStates(page: page, locator: locator, st
 test('Assert "Add voters" button is only visible to owner', async ({ page }) => {
   await assertIsOnlyVisibleToOwnerStates(page, page.getByTestId('addVotersButton'), [0, 1]);
 });
+
+test('Assert "Initialize" button is only visible to owner', async ({ page }) => {
```

Comment:
Lazy review: are there positive tests on the buttons where it checks that the buttons are available?

Reply:
not yet, that's why it closes #80 and not #72 :slightly_smiling_face:

Reply:
or at least not all of them - e.g. it checks that the buttons that are should only be visible to owner are:

* not visible to unauthenticated
* not visible to non-owner user
* not visible to non-owner admin
* visible to owner

but it does not e.g. check that the "Vote" button is visible for voters

Reply:
also it checks only in specific states, e.g. it does not verify that the buttons are not displayed in other states where they are not applicable at all